src/service/referral-api/src/FamilyHubs.Referral.Data/Migrations/20240927164033_AddOneToManyRelationshipBetweenOrganisationAndReferralServiceAndRemoveOneToOneRelationship.cs

```diff
@@ -0,0 +1,82 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace FamilyHubs.Referral.Data.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddOneToManyRelationshipBetweenOrganisationAndReferralServiceAndRemoveOneToOneRelationship : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropForeignKey(
+                name: "FK_Organisations_ReferralServices_ReferralServiceId",
+                table: "Organisations");
+
+            migrationBuilder.DropIndex(
+                name: "IX_Organisations_ReferralServiceId",
+                table: "Organisations");
+
+            migrationBuilder.DropColumn(
+                name: "ReferralServiceId",
+                table: "Organisations");
+
+            migrationBuilder.AddColumn<long>(
+                name: "OrganizationId",
+                table: "ReferralServices",
+                type: "bigint",
+                nullable: false,
+                defaultValue: 0L);
+
+            migrationBuilder.CreateIndex(
+                name: "IX_ReferralServices_OrganizationId",
+                table: "ReferralServices",
+                column: "OrganizationId");
+
+            migrationBuilder.AddForeignKey(
```

Comment:
On a fresh db this migration fails here as the default value (0) does not match any organisations in the db.
(A previous migration adds the example Elop Mentoring ReferralService)